docs/updating/nim-migration/custom-plugins.md

```diff
@@ -9,4 +9,141 @@ modified_at: 2024-06-06
 
 # Migrating custom plugins
```

Comment:
I would really make it more specific to plugins that were published as libraries. 99% of people don't need this quide, and it would be best to not confuse them.

Reply:
Only 1% or less of framework devs publish their plugins.

Reply:
Done :heavy_check_mark: